src/foam/u2/view/DateView.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'DateView',
+  extends: 'foam.u2.view.ModeAltView',
+
+  documentation: 'A view for foam.core.Date properties.',
+
+  requires: [
+    'foam.u2.DateView',
+    'foam.u2.view.ValueView'
+  ],
+
+  properties: [
+    {
+      name: 'readView',
+      value: { class: 'foam.u2.view.ValueView' }
```

Comment:
Would it make sense for this to be the default value for ModeAltView.readView?

Reply:
I would argue that there shouldn't be default views for `ModeAltView`. It's essentially a combinator, so any two views will work. To give one of them a default value related to primitive properties might give the impression that `ModeAltView` is in some sense specific to primitive properties, whereas without the defaults it's more clear that it's simply a combinator.

Reply:
Then would something like an AnyReadView be a good idea? (for a future PR, possibly), which really only just handled primitives and FObjects (as Detail or CitationViews)?

Reply:
If all of the primitive types and FObjectProperty already have correct read views by default (as set by this PR), then is there any point in adding AnyReadView? I don't see the use case for it.